src/SerialLoops.Lib/Items/ItemDescription.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SerialLoops.Lib.Items
+{
+    public class ItemDescription
+    {
+
+        public string Name { get; private set; }
+        public ItemType Type { get; private set; }
+
+        public ItemDescription(string name, ItemType type)
+        {
+            Name = name;
+            Type = type;
+        }
+
+        // Load an item from an ItemDescription
+        public Item Load()
```

Comment:
I'm guessing we want to override this in future classes, in which case it'd be

```suggestion
        public virtual Item Load()
```